### PR TITLE
[8.0][FIX] Travis and Runbot

### PR DESCRIPTION
**Pull request by thomaspaulb**

### CODE REVIEW

File: l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def test_l10n_nl_xaf_auditfile_export_default(self):
         zf = StringIO(b64decode(self.export.auditfile))
         with ZipFile(zf, 'r') as archive:
             contents = archive.read(archive.namelist()[0])
-        self.assertTrue(contents.startswith('<?xml '))
```
Comment (NL66278) [inline l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py:34]: Why these changes?

Comment (thomaspaulb) [reply l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py:34]: I see that the `l10n_nl_xaf_auditfile_export` test is failing on this line, but I cannot see what it generates instead of `<?xml `. And on my local the test is passing. An assertEquals would show me directly in Travis what is generated instead, so that I can fix it.

Comment (NL66278) [reply l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py:34]: OK, thanks for the explanation!

File: l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def _compute_auditfile_name(self):
             item.auditfile_name = '%s.xaf' % item.name
             if item.auditfile:
                 auditfile = base64.b64decode(item.auditfile)
-                zf = StringIO(auditfile)
+                zf = BytesIO(auditfile)
```
Comment (thomaspaulb) [inline l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py:46]: This is now in line with [10.0](https://github.com/OCA/l10n-netherlands/blob/10.0/l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py#L42)

File: l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py [hunk 3]
```diff
@@ -28,16 +28,18 @@ def setUp(self):
...
         self.export.write({'data_export': 'all'})
         self.export.button_generate()
         self.assertTrue(self.export.auditfile)
-        zf = StringIO(b64decode(self.export.auditfile))
+        zf = BytesIO(b64decode(self.export.auditfile))
         with ZipFile(zf, 'r') as archive:
-            contents = archive.read(archive.namelist()[0])
-        self.assertTrue(contents.startswith('<?xml '))
+            filelist = archive.filelist
+            contents = archive.read(filelist[-1]).decode('utf8')
```
Comment (thomaspaulb) [inline l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py:44]: Also now in line with [10.0](https://github.com/OCA/l10n-netherlands/blob/10.0/l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py#L42).

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Comment by thomaspaulb** [comment]
@hbrunn I see you are a PSC here.. could I look at you nicely?
